src/UIModule.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module;
+
+use Dhii\Data\Container\ContainerFactoryInterface;
+use Psr\Container\ContainerInterface;
+use Psr\EventManager\EventManagerInterface;
+use RebelCode\Modular\Module\AbstractBaseModule;
+
+class UIModule extends AbstractBaseModule

```

Comment:
As much as possible, try to keep the module class name similar to the module package and repo name. Also, stick to camel case. In this case, I would call this class `WpBookingsUiModule`.